web-ui/view/build.ml

```diff
@@ -54,6 +54,12 @@ let title_card ~status ~card_title ~hash_link ~ref_links ~first_created_at
           ];
       ])
 
+let tabulate rows =
+  Tyxml.Html.(
+    div
+      ~a:[ a_class [ "container-fluid mt-8 flex flex-col space-y-6" ] ]
+      [ div ~a:[ a_class [ "table-container" ] ] rows ])
```

Comment:
I don't think this is correct -- I do not see a `table-container` class in the designer's sample of the Refs page. Also, this function is not relevant to the Build page so I suggest it should not appear here ... if it is a common function to various views it could go into `common.ml`

Reply:
Also, I think that the added `table-container` class is throwing off the css. Please can we review the markup that the designer gave us - I think we are missing a div with classes 

```
border border-gray-200 rounded-lg w-full overflow-hidden shadow-sm  divide-y divide-gray-200
```